project_event/models/project_project.py

```diff
@@ -30,6 +30,30 @@ class Project(models.Model):
     notes = fields.Text(
         string='Notes',
     )
+    state = fields.Selection([
+        ('draft', 'Draft'),
+        ('option', 'Option'),
+        ('requested', 'Requested'),
+        ('read', 'Read'),
+        ('accepted', 'Accepted'),
+        ('done', 'Done'),
+        ('canceled', 'Canceled')],
+        string='Task State',
```

Comment:
@eilst ;"], " bring in the new line .
project_project.py mostly handle for event information and why you define the string as string='Task State',
Please order the selection fields type based on chronological order

Reply:
It's already on chronological order. I guess you meant alphabetical order, but in this case it's better chronological order